api/ServiceWorkerRegistration.json

```diff
@@ -435,16 +435,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/periodicSync",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": false
```

Comment:
This was [removed in Chrome 48](https://storage.googleapis.com/chromium-find-releases-static/7b4.html#7b48be9441eec3f2373f68f289b6dce85c6402f8). I would prefer showing addition and removal here instead of just `false`. The former shows that the feature was tried and rejected whereas the latter might suggest to an unknowledgeable reader that the feature could be added some time in the future.